From cf6fd4e849ca9038a94b7c73d3c5ac2e208a8dfc Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sun, 17 Dec 2023 07:58:34 -0800
Subject: [PATCH 2/2] VVI-node-api-headers : node-api-headers BUILD file

Summary:
Add BUILD file for bzlmod registry

Test Plan:
integration with vvplatform
---
 BUILD | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 BUILD

diff --git a/BUILD b/BUILD
new file mode 100644
index 0000000..530cf2e
--- /dev/null
+++ b/BUILD
@@ -0,0 +1,10 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "node-api-headers",
+    hdrs = glob(["include/*.h*",]),
+    includes = [
+        "include",
+    ],
+    visibility = ["//visibility:public"],
+)
-- 
2.34.1

